mlfinlab/codependence/information_matrix.py

```diff
@@ -0,0 +1,67 @@
+"""
+This script implements the creation of dependence and distance matrix for the
```

Comment:
We should elaborate on the type of distance and why plus some paper references. re our conversation on slack, I highlighted another distance metric which is found in the literature but different to this.